tel2puml/find_unique_graphs/otel_ingestion/json_data_converter.py

```diff
@@ -819,7 +821,9 @@ def process_spans(
 
     if isinstance(data, list):
         if not data:
-            raise ValueError("Spans list is empty")
+            LOGGER.warning(
+                "WARNING - Encountered an empty list whilst processing spans."
```

Comment:
Don't need to put WARNING in the text as on screen it will show log level i.e. WARNING

Reply:
Removed